### PR TITLE
2: Minimum prototype for pilot

### LINKED ISSUE #2: Minimum prototype functionality to support pilot application
Idea: take `tern.mmrm` as starting point and then adapt to `tern.gee`, since here we also have visits and covariance structures etc.

Includes: 
- [x] different family/link are possible - work with 2 class levels : 1) GEE and 2) GEE.logistic e.g.
- [x] make it less automatic re: inclusion of arm*visit interaction?
- [x] tabulation of fixed effects and covariance matrix estimates
- [x] LS means:
	- [x] make the LS means afterwards and not as part of the initial fit
	- [x] tabulation specific per family, e.g. for GEE.logistic only for the beginning
- [x] switch to using the `geeasy` package

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #2

### CODE REVIEW

**Comment by danielinteractive** [comment]
@cicdguy we still get the `nestcolor` staged dependencies problem:
image
I already tried rerunning the checks but that does not help. any idea?

**Comment by walkowif** [comment]
Looks like `nestcolor` was not added to `DESCRPTION`. After I added it, staged dependencies pass.

File: .github/workflows/check.yaml [hunk 1]
```diff
@@ -24,8 +24,6 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
     secrets:
       REPO_GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
-    with:
-      additional-r-cmd-check-params: --as-cran
```
Comment (shajoezhu) [inline .github/workflows/check.yaml:28]: Hi @edelarua, can you reinstate these two lines please

File: DESCRIPTION [hunk 2]
```diff
@@ -13,22 +14,19 @@ URL: https://github.com/insightsengineering/tern.gee/
...
-    tern (>= 0.7.9),
-    nestcolor (>= 0.0.1)
+    tern (>= 0.7.9)
 Imports:
-    checkmate,
-    geeasy,
-    geepack,
-    nlme,
-    rtables,
-    stats
```
Comment (shajoezhu) [inline DESCRIPTION]: Hi @edelarua, can you reinstate these lines please

File: NAMESPACE [hunk 3]
```diff
@@ -6,6 +6,7 @@ S3method(as.rtable,tern_gee)
 S3method(lsmeans,tern_gee_logistic)
 export(a_lsmeans_logistic)
 export(as.rtable)
+export(dummy_function)
```
Comment (shajoezhu) [inline NAMESPACE:9]: for removal

File: R/hello.R [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#' A dummy function
+#' @export
+dummy_function <- function() {
+  rtables::basic_table()
+  formatters::divider_height()
+  as.rtable(data.frame(x = 1:10))
+}
```
Comment (shajoezhu) [inline R/hello.R:7]: for removal

File: tern.gee.Rproj [hunk 5]
```diff
@@ -1,7 +1,7 @@
 Version: 1.0
 
-RestoreWorkspace: No
-SaveWorkspace: No
+RestoreWorkspace: Default
+SaveWorkspace: Default
```
Comment (shajoezhu) [inline tern.gee.Rproj:4]: No instead of Default

File: tern.gee.Rproj [hunk 6]
```diff
@@ -14,8 +14,8 @@ LaTeX: pdfLaTeX
 
 AutoAppendNewline: Yes
 StripTrailingWhitespace: Yes
+LineEndingConversion: Posix
```
Comment (shajoezhu) [inline tern.gee.Rproj:17]: for removal

File: tests/testthat/test-dummy.R [hunk 7]
```diff
@@ -0,0 +1,6 @@
+test_that("dummy", {
+  # Compare
+  result <- "Hello, Tim"
+  expected <- "Hello, Tim"
+  expect_identical(result, expected)
+})
```
Comment (shajoezhu) [inline tests/testthat/test-dummy.R:6]: for removal

**Review by shajoezhu — CHANGES_REQUESTED** [review]
Hi @edelarua , can you please help with these changes please. when I did the hotfix on main, i used some dummy function and tests. lets remove them plesae. Thanks!

**Comment by edelarua** [comment]
> Hi @edelarua , can you please help with these changes please. when I did the hotfix on main, i used some dummy function and tests. lets remove them plesae. Thanks!

All complete now.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,77 @@
 # tern.gee
 
-Create TLGs using Generalized Estimating Equations (GEE)
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
```
Comment (danielinteractive) [inline README.md:3]: maybe consider using "work in progress" status instead of "active"?

Comment (edelarua) [reply README.md]: Done!

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @edelarua and @danielinteractive 
I think this is almost ready. approve this PR to unblock work as I am ooo this week. Thanks!

**Comment by shajoezhu** [comment]
hi @edelarua , could you resolve the conflict on this branch please. Thanks!

**Comment by edelarua** [comment]
@danielinteractive @shajoezhu All checks are passing now except for the staged.dependencies check. I see that Dinakar opened a PR a while ago to add `nestcolor` as an upstream dependency for `tern.gee` (https://github.com/insightsengineering/nestcolor/pull/36). Is this a dependency we still want to include?

**Comment by shajoezhu** [comment]
Hi @edelarua , I think we can keep nestcolor out for tern.gee for now. It is not doing any plotting, we can keep these dependency tree a bit simpler. Thanks!

**Pull request merged**